Fix feed by not using pagination

frankieroberto opened the pull request:
I think this should resolve #330 but need to test it somehow.

paulrobertlloyd commented:
Would something like this in your `package.json` work for testing:

```json
dependencies: {
  "@x-govuk/govuk-eleventy-plugin": "x-govuk/govuk-eleventy-plugin#fix-feed-by-not-using-pagination"
}
```

frankieroberto commented:
@paulrobertlloyd hmm I can’t work this out at all. I’ve removed slice (which doesn’t do what I thought it did in Nunjucks) and replaced it with basic `if` condition, and removed the sort, and all the date outputs, and I’m still getting the same `TypeError: Cannot read properties of undefined (reading 'toISOString')` error.  Can't quite work out where this is coming from...

The official RSS plugin for Eleventy 3.0 might well be a better solution all round!

paulrobertlloyd commented:
Similar fix committed in 9e5467b2601263ee0b1149143508e6fafa0dc6c0, will ship as part of v7.0
